packages/commonjs/src/transform-commonjs.js

```diff
@@ -70,6 +72,7 @@ export default function transformCommonjs(
   const HELPERS_NAME = deconflict(scope, globals, 'commonjsHelpers');
   const namedExports = {};
   const dynamicRegisterSources = new Set();
+  let removedRequireCount = 0;
```

Comment:
As we are not really interested in the count, we could also make this just a boolean, e.g. `hasRemovedRequires = false` and later just set it to `true`.

Reply:
I remember wanting to do something with it, but forgot about it. No idea what it was. Converted to boolean